### PR TITLE
core: export plugin constant.

### PR DESCRIPTION
**Pull request by Txiaozhe**

### CODE REVIEW

File: packages/pipcook-core/src/index.ts [hunk 1]
```diff
@@ -9,4 +9,5 @@ export {PipcookRunner} from './core/core';
 export {createAnnotationFile, parseAnnotation, unZipData, downloadZip, getOsInfo, transformCsv,
   getDatasetDir, createAnnotationFromJson, getMetadata, getModelDir, convertPascol2CocoFileOutput, compressTarFile } from './utils/publicUtils';
 export {startBoard} from './board/board';
-export {PipcookComponentResult} from './types/component';
\ No newline at end of file
+export {PipcookComponentResult} from './types/component';
+export {PLUGINS} from './constants/plugins';
```
Comment (yorkie) [inline packages/pipcook-core/src/index.ts]: We should export `constants` only at the top-level just like `constants.PLUGINS` :)

Comment (Txiaozhe) [reply packages/pipcook-core/src/index.ts]: Yes I did, please take a look:)

File: packages/pipcook-core/src/index.ts [hunk 2]
```diff
@@ -9,4 +9,14 @@ export {PipcookRunner} from './core/core';
...
+export {
+  DATACOLLECT,
+  DATAACCESS,
+  DATAPROCESS,
+  MODELLOAD,
+  MODELTRAIN,
+  MODELEVALUATE,
+  MODELDEPLOY,
+  ONLINETRAIN
+} from './constants/plugins';
```
Comment (yorkie) [inline packages/pipcook-core/src/index.ts]: What I meant is:

```ts
import {PLUGINS} from './constants/plugins';
...
export constants = {
  PLUGINS,
};
```

Comment (Txiaozhe) [reply packages/pipcook-core/src/index.ts]: awkward:(

Comment (yorkie) [reply packages/pipcook-core/src/index.ts]: Never mind :p

**Pull request merged**